raytracing/eo.py

```diff
@@ -12,7 +12,7 @@ class PN_33_922(AchromatDoubletLens):
     def __init__(self):
         # PN for Part number
         super(PN_33_922,self).__init__(fa=150.00,fb=126.46, R1=92.05,R2=-72.85, R3=-305.87, 
-                                    tc1=23.2, tc2=23.1, te=36.01, n1=1.6700, n2=1.8467, diameter=75,
+                                    tc1=23.2, tc2=23.1, te=36.01, n1=N_BAK1.n(0.5876), n2=N_SF8.n(0.5876), diameter=75,
```

Comment:
I found the problem on Edmund Optics web site, see commit https://github.com/DCC-Lab/RayTracing/commit/d026dabe2f600cb9ec4c39266814acbb73e3a9d3

Reply:
I thought the composing glasses were N-BAF10 and N-SF57? When I looked up for those, I found the same n1 and n2 as the values before the change. I'd be interested to know where you found the new values?

Reply:
As mentionned in the commit https://github.com/DCC-Lab/RayTracing/commit/d026dabe2f600cb9ec4c39266814acbb73e3a9d3, this file is the actual "design reference": 
https://www.edmundoptics.com/document/download/405650
Note 1, at the top left, says the actual material used in the modelling.